Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
+        if not key:
+            tkMessageBox.showerror(title=self.keyerror_title, parent=self,
+                                   message="No keys specified.")
```

Comment:
Three possible updates:
from tkinter import messagebox
from tkinter.messagebox import showerror (for instance)
<replace showerror with custom function, #30751